capsules/src/bmp280.rs

```diff
@@ -73,7 +73,7 @@ impl CalibrationData {
         }
     }
 
-    fn temp_from_raw(&self, raw_temp: u32) -> i32 {
+    fn temp_from_raw(&self, raw_temp: i32) -> i32 {
```

Comment:
```suggestion
    fn temp_from_raw(&self, temp: i32) -> i32 {
```

The following line can be removed with the above change.